.idea/BNZ-P2P-Lending.iml

```diff
@@ -8,5 +8,6 @@
     </content>
```

Comment:
I think these files under `.idea/` are from the IDE. I think these should generally be removed from a repository (as different people may be using different dev environments). So they're just clutter.